redis_enterprise/assets/dashboards/redis-enterprise-database-dashboard-v2.json

```diff
@@ -0,0 +1,2 @@
height":2}},{"id":2965764631170102,"definition":{"title":"Memory Limit","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:rdse.bdb_memory_limit{$cluster,$database}","aggregator":"last"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2,"timeseries_background":{"type":"area","yaxis":{"include_zero":true}}},"layout":{"x":0,"y":6,"width":2,"height":2}},{"id":6841829436079466,"definition":{"title":"Fork CPU","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"avg:rdse.bdb_fork_cpu_system{$cluster,$database}","aggregator":"last"}],"formulas":[{"formula":"query1"}]}],"autoscale":false,"precision":2,"timeseries_background":{"type":"area"}},"layout":{"x":2,"y":6,"width":2,"height":2}},{"id":6105475889986624,"definition":{"title":"Evicted Objects","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"avg:rdse.bdb_evicted_objects{$cluster,$database}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2,"timeseries_background":{"type":"area"}},"layout":{"x":4,"y":6,"width":2,"height":2}},{"id":208920565392582,"definition":{"title":"Expired Objects","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"avg:rdse.bdb_expired_objects{$cluster,$database}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2,"timeseries_background":{"yaxis":{"include_zero":true},"type":"area"}},"layout":{"x":6,"y":6,"width":2,"height":2}},{"id":5190993737261022,"definition":{"title":"Instantaneous Ops","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"avg:rdse.bdb_instantaneous_ops_per_sec{$cluster,$database}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2,"timeseries_background":{"type":"area"}},"layout":{"x":8,"y":6,"width":2,"height":2}},{"id":2052381121377014,"definition":{"title":"Total Keys","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"max:rdse.bdb_no_of_keys{$cluster,$database}","aggregator":"last"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2,"timeseries_background":{"type":"area"}},"layout":{"x":10,"y":6,"width":2,"height":2}}],"template_variables":[{"name":"cluster","prefix":"cluster","available_values":[],"default":"*"},{"name":"database","prefix":"bdb_name","available_values":[],"default":"*"}],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed","tags":[]}
```

Comment:
Dashboard is missing a description, see [best practices guidelines](https://docs.datadoghq.com/developers/integrations/create-an-integration-dashboard?tab=ui#follow-dashboard-best-practices).